README.md

```diff
@@ -25,9 +26,9 @@ This role has been generated using the [cookiecutter](https://github.com/cookiec
 These instructions will get you a copy of the role for your ansible playbook. Once launched, it will install an [Prometheus Apache Exporter](https://github.com/Lusitaniae/apache_exporter) server in a Debian system.
 
 ### Prerequisities
-Ansible > 2.8.0 version installed.
+Ansible 4.6.0 version installed.
```

Comment:
Should we update the min_ansible_version from the meta/main.yml in order to reflect those changes?

Reply:
Done :)